<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbView</class>
 <widget class="QDialog" name="DbView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="dbView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>31</y>
     <width>461</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="helper">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Click Item Code to Edit</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>131</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="connectMongo">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CreateItem">
      <property name="text">
       <string>New</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="connectStatus">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>191</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
